pwiz_tools/Skyline/Model/RefinementSettings.cs

```diff
@@ -1025,7 +1025,8 @@ public enum ConvertToSmallMoleculesChargesMode
                         Resources.RefinementSettings_ConvertToSmallMolecules_Converted_To_Small_Molecules,

```

Comment:
Let's go over this section. I still see a mix of "document" and "newDoc" which doesn't look completely rationalized to me.